CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+
+When reporting a bug, please include: 
+- A clear and descriptive title for the issue
+- Your operating system and its version
+- The version of Oríon (even if you're using the latest!)
+- A description of the bug
+- The steps to reproduce the bug
+- Extra details about your setup that might help us to reproduce the bug if not included previously
+
+### How to propose enhancements
+We're thrilled to hear you found a way to make Oríon better through minor improvements to completely new features! 
+Before creating enhancement suggestions, please check that your idea is not already present in the list of issues. You might find that you don't need to create one and can just join in the discussion directly and give your opinion!
+
+When suggesting enhancements, it is a good idea to:
+- Write a clear and descriptive title for the issue
+- Motivate why this feature is relevant regarding [Oríon's mission](https://github.com/Epistimio/orion).
+- Explain in detail how it would work.
+- Keep the scope as narrow as possible, to make it easier to implement.
+- Add as much details as possible!
+
+### How to submit changes
+We're grateful you're considering making changes to Oríon! To get started, you need to first fork the repository and then create a new branch on your fork where you'll do your changes. Once you implemented the changes, submit a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests>). All changes to Oríon are done through PRs, where they will be peer-reviewed and checked against our continuous integration system to ensure the quality of the code base.
+
+During this processs keep in mind to:
+- Set a descriptive and short branch name
+- Create a descriptive and clear title for the PR
+- Motivate the *why* and *how* of the PR (get inspiration from current or past PRs!)
+- Write [good commits](https://chris.beams.io/posts/git-commit/)
+- Include tests for new features or bug fixes
+- Update the source code documentation (docstring) and, if applicable, the [general documentation](https://orion.readthedocs.io/en/latest/index.html)
+
+Next, you need to get familiar with the developer documentation. You will find the instructions to set up your development environment, including how to use the test suite and verify you changes will pass the CI as well as our style guides for source code and documentation.
+
+-> https://orion.readthedocs.io/en/latest/developer/overview.html
+
+#### Your first contribution
+If you are not sure about what to work on first, we suggest you take a look on the [help wanted](https://github.com/Epistimio/orion/labels/help%20wanted) and [good first issues](https://github.com/Epistimio/orion/labels/good%20first%20issue) opened. They are great places to start!
+
+---
+Thank you for contributing! We really appreciate it!
+
+-- The Oríon team
```

Comment:
There isn't really an `Oríon team`. :thinking: I would tend to leave this unsigned.